scripts/Dockerfile

```diff
@@ -1,18 +1,31 @@
-FROM jupyter/minimal-notebook
+FROM jupyter/base-notebook:python-3.9.12
```

Comment:
Good call on switching to `base-notebook`.  Any reason to fix the python version?  I think it might be preferable to have the container test build on "latest", just so no one needs to remember to bump versions.

Reply:
Yes, using `base-notebook` saves some space. The image with a specific python 3.9 version will make it consistent to the python version in [environment-dev-py39.yml](https://github.com/pymc-devs/pymc/blob/main/conda-envs/environment-dev-py39.yml), which will be the only dev environment left after https://github.com/pymc-devs/pymc/pull/5842 will be merged. This image is also updated regularly on Dockerhub I think. 

It also work with latest `base-notebook`. I have tested in before, however the python version will be 3.10 (if I am not wrong), and our tests seem not ready with that yet https://github.com/pymc-devs/pymc/issues/5209 

I am happy to change it to the `latest` `base-notebook` image if you guys prefer this option.